app/services/candidate_interface/inactive_date_calculator.rb

```diff
@@ -1,36 +1,38 @@
 module CandidateInterface
   class InactiveDateCalculator
-    def initialize(application_choice:, effective_date:, time_limit_calculator: TimeLimitCalculator)
-      @application_choice = application_choice
-      @time_limit_calculator = time_limit_calculator.new(rule: :reject_by_default, effective_date:)
+    STANDARD_DAYS_UNTIL_INACTIVE = 30
+
+    def initialize(effective_date:)
```

Comment:
The `application_choice` was not used in the class anymore 
The `TimeLimitCalculator` was... not necessary and impossible to read.